pyproject.toml

```diff
@@ -22,7 +22,7 @@ python = "^3.8"
 marshmallow = "3.13.0"
 marshmallow-dataclass = "8.5.3"
 click = "8.0.1"
-PyYAML = "5.4.1"
+PyYAML = "6.0.1"
```

Comment:
There was an issue with cython from last week affecting our version, options were to downgrade or upgrade